iroh/src/node/builder.rs

```diff
@@ -85,7 +94,7 @@ where
     gc_policy: GcPolicy,
     dns_resolver: Option<DnsResolver>,
     node_discovery: DiscoveryConfig,
-    docs_store: iroh_docs::store::Store,
+    docs_store: Option<DocsStorage>,
```

Comment:
why not just add a `Disabled` variant to the enum, this way you don't need to interpret the `Option`

Reply:
changed to that!